packages/gatsby-theme-newrelic/src/components/ExternalLink.js

```diff
@@ -3,26 +3,40 @@ import PropTypes from 'prop-types';
 import useInstrumentedHandler from '../hooks/useInstrumentedHandler';
 import { localizeExternalLink } from '../utils/localization';
 import useLocale from '../hooks/useLocale';
+import useTessen from '../hooks/useTessen';
+import { useLocation } from '@reach/router';
 
 const isNewRelic = (to) => to.startsWith('https://newrelic.com');
+const isSignup = (to) => to.startsWith('https://newrelic.com/signup');
 
-const ExternalLink = ({ children, onClick, href, ...props }) => {
+const ExternalLink = ({ children, onClick, href, component, ...props }) => {
   const locale = useLocale();
+  const tessen = useTessen();
+  const location = useLocation();
+
+  const handleClick = useInstrumentedHandler(onClick, {
+    actionName: 'externalLink_click',
+    href: href,
+  });
 
   const link = isNewRelic(href)
     ? localizeExternalLink({ link: href, locale })
     : href;
 
-  const handleClick = useInstrumentedHandler(onClick, {
-    actionName: 'externalLink_click',
-    href: link,
-  });
+  const trackSignUp = () => {
+    handleClick();
```

Comment:
[nit] Can you pass through the `event` to this click handler? While most of the time we don't use it, we should forward it in case something does need to use it (like a `preventDefault` or something).

Reply:
yep!